src/EtherFiAdmin.sol

```diff
@@ -153,7 +159,7 @@ contract EtherFiAdmin is Initializable, OwnableUpgradeable, UUPSUpgradeable {
         numValidatorsToSpinUp = _report.numValidatorsToSpinUp;
 
         _handleAccruedRewards(_report);
-        _handleValidators(_report, _pubKey, _signature);
+        _handleValidators(reportHash, _report, _pubKey, _signature);
```

Comment:
should we remove _pubKey in _handleValidators waste of massive argument. It does nothing but get hashed in the _enqueueAdminTask_ApproveValidators. The benefit is that during the same pubkeys are provided as were passed during the executeTask, but if thats wrong then this is wrong as well.